File: app/src/main/java/com/wealoha/social/beans/search/FindYouResult.java
package com.wealoha.social.beans.search;

import java.util.List;
import java.util.Map;

import com.wealoha.social.beans.ResultData;
import com.wealoha.social.beans.User;

public class FindYouResult extends ResultData {

	public Map<String, String> highlightMap;
	public List<User> list;
	public User user;
	public String keyword;
}

File: app/src/main/java/com/wealoha/social/beans/search/FindYouResult.kt
package com.wealoha.social.beans.search

import com.wealoha.social.beans.ResultData
import com.wealoha.social.beans.User

class FindYouResult : ResultData() {
    @kotlin.jvm.JvmField
    var highlightMap: Map<String, String>? = null
    @kotlin.jvm.JvmField
    var list: List<User>? = null
    @kotlin.jvm.JvmField
    var user: User? = null
    @kotlin.jvm.JvmField
    var keyword: String? = null
}